lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
@remear from the link in this PR description

> A “relationship object” MUST contain at least one of the following:

And then specifies `data`, `links`, and `meta`.

> I'm not following. If the GET returns null in the example you gave, there would be no actual relationship persisted in the system, right?

The relationship is persisted, we just aren't returning it in this response. If you return data `null`, you are saying "this relationship doesn't exist" versus "this API response does not contain info about this relationship"